scipy/stats/stats.py

```diff
@@ -6937,12 +6937,12 @@ def kstest(rvs, cdf, args=(), N=20, alternative='two-sided', mode='auto'):
     Performs the (one sample or two samples) Kolmogorov-Smirnov test for goodness of fit.
 
     The one-sample test performs a test of the distribution F(x) of an observed
-    random variable against a given distribution G(x). Under the null
-    hypothesis, the two distributions are identical, F(x)=G(x). The
-    alternative hypothesis can be either 'two-sided' (default), 'less'
-    or 'greater'. The KS test is only valid for continuous distributions.
-    The two-sample test tests whether the two independent samples are drawn
-    from the same continuous distribution.
+    random variable against a given distribution G(x).
+
+    The two-sample test `ks_2samp` tests whether the two independent samples
```

Comment:
what about: 

The one-sample test performs a test of the distribution F(x) of an observed random variable against a given distribution G(x). The two-sample test `ks_2samp` compares the distribution of two independent samples. 

(same distribution sounds like F=G, I am just trying to avoid that statement)

Reply:
I see. I applied this. Can we also use the same structure/language in the two sentences to make the similarities and differences stand out?

The one-sample test compares the underlying distribution F(x) of a sample against a given distribution G(x). The two-sample test `ks_2samp` compares the underlying distributions of two independent samples.

Reply:
sounds good!